packages/cli/src/commands/init/init.js

```diff
@@ -58,7 +58,7 @@ async function setProjectDirectory(directory) {
       throw new DirectoryAlreadyExistsError(directory);
     }
 
-    await fs.emptyDir(directory);
+    fs.emptyDirSync(directory);
```

Comment:
I wonder, why this change?

Reply:
Using await causes hang on Windows, that's why.

Reply:
Geez, Windows is so weird

Reply:
It indeed is. 😂😂